index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div class="landingPage">
+        <header>
+            <div class="logo">My Logo</div>
+            <div class="menu">
+                <img src="./images/menu.svg" class="menuIcon" alt="">
+            </div>
```

Comment:
- [ ] [REQUIRED] Please implement my suggestions below:

  * Please, your `logo` should be on the left (✔️ ) and has an `a` link (`a` tag). This is for three reasons: `convention`, `semantics`, and `user` experience(`UX`). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info 👍🏼 

  * Please change the `div` tag to the `nav` tag and header elements should be inside of the `nav` tag because the `nav` tag is for semantics reasons and is the right position (✔️) for convention and `user` experience for mobile devices 😃 

```html
<header>
    <nav>
        <a>Website Logo</a>
        <!-- Main Navigation inside <header> -->
        <ul>
            <li></li>
        </ul>
    </nav>
</header>
```